catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/ScriptFragment.kt

```diff
@@ -0,0 +1,1009 @@
+        }
+        when (actionModeType) {
+            BACKPACK -> showNewScriptGroupAlert(adapter.selectedItems)
+            COPY -> copy(adapter.selectedItems)
+            DELETE -> showDeleteAlert(adapter.selectedItems)
+            COMMENT -> toggleComments(adapter.selectedItems)
+            NONE -> throw IllegalStateException("ActionModeType not set correctly")
+            CATBLOCKS -> {
+            }
+        }
+    }
+
+    private fun resetActionModeParameters() {
+        actionModeType = NONE
+        actionMode = null
+        adapter.setCheckBoxMode(BrickAdapter.NONE)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        val view = View.inflate(activity, R.layout.fragment_script, null)
+        listView = view.findViewById(android.R.id.list)
+        scriptFinder = view.findViewById(R.id.findview)
+
+        scriptFinder?.setOnResultFoundListener(object : OnResultFoundListener {
+            override fun onResultFound(
+                sceneIndex: Int,
+                spriteIndex: Int,
+                brickIndex: Int,
+                totalResults: Int,
+                textView: TextView?
+            ) {
+                val currentProject =
+                    projectManager.currentProject
+                val currentScene =
+                    currentProject.sceneList[sceneIndex]
+                val currentSprite =
+                    currentScene.spriteList[spriteIndex]
+                textView?.text = createActionBarTitle(
+                    currentProject,
+                    currentScene,
+                    currentSprite
+                )
+                projectManager.setCurrentSceneAndSprite(
+                    currentScene.name,
+                    currentSprite.name
+                )
+                adapter.updateItems(currentSprite)
+                adapter.notifyDataSetChanged()
+                listView?.smoothScrollToPosition(brickIndex)
+                highlightBrickAtIndex(brickIndex)
+                hideKeyboard()
+            }
+        })
+
+        val activity = activity as SpriteActivity?
+
+        scriptFinder?.setOnCloseListener(object : OnCloseListener {
+            override fun onClose() {
+                listView?.cancelHighlighting()
+                finishActionMode()
+                if (activity != null && !activity.isFinishing) {
+                    activity.setCurrentSceneAndSprite(
+                        projectManager.currentlyEditedScene,
+                        projectManager.currentSprite
+                    )
+                    activity.supportActionBar?.title = activity.createActionBarTitle()
+                    activity.addTabs()
+                }
+                activity?.findViewById<View>(R.id.toolbar)?.visibility = View.VISIBLE
+            }
+        })
+
+        scriptFinder?.setOnOpenListener(object : OnOpenListener {
```

Comment:
please convert to lambda.
```suggestion
        scriptFinder?.setOnOpenListener {
```

Reply:
![image](https://user-images.githubusercontent.com/46495964/125193394-d94bd080-e269-11eb-97e0-32d92132215d.png)